packages/plugins/integration-hubstaff-ui/src/lib/components/hubstaff-authorize/hubstaff-authorize.component.ts

```
@@ -0,0 +1,160 @@
+		private readonly _hubstaffService: HubstaffService,
+		private readonly _fb: UntypedFormBuilder,
+		private readonly _router: Router,
+		private readonly _store: Store,
+		private readonly _integrationsService: IntegrationsService
+	) {}
+
+	ngOnInit() {
+		this._store.selectedOrganization$
+			.pipe(
+				filter((organization) => !!organization),
+				tap((organization: IOrganization) => (this.organization = organization)),
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)